src/main/java/org/aion/wallet/ui/components/HistoryController.java

```diff
@@ -127,6 +151,28 @@ private void setEventHandlers() {
         copyItem.setOnAction(new ContextMenuTableCopyEventHandler(txTable));
         menu.getItems().add(copyItem);
         txTable.setContextMenu(menu);
+
+
+        searchField.textProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
we should add this listener to the drop-down field as well, albeit it will probably have a different filtering predicate